### PR TITLE
docs: update image style guide doc for aliased images

### PR DESCRIPTION
**Pull request by roadlittledawn**
## Description

Updates style guide doc on adding images to files for aliased images

## Related issues / PRs
partially closes https://github.com/newrelic/docs-website/issues/6613

### CODE REVIEW

File: src/content/docs/style-guide/images/embed-images.mdx [hunk 1]
```diff
@@ -15,61 +15,114 @@ redirects:
   - /docs/nronly/images
   - /docs/style-guide/processes-procedures/edit-images
   - /docs/style-guide/processes-procedures/embed-images
+resources:
+  - title: Image codemod README
+    url: 'https://github.com/newrelic/docs-website/blob/b43d853f9ba4b17d4c090fb0a650b8f18cb0841d/codemods/convertMDXImages/README.md'
```
Comment (roadlittledawn) [inline src/content/docs/style-guide/images/embed-images.mdx:20]: i'll eventually update this to point to the README in `develop`

**Comment by roadlittledawn** [comment]
sorry for all the `prettier` formatting changes 😞

File: src/content/docs/style-guide/images/embed-images.mdx [hunk 2]
```diff
@@ -15,61 +15,114 @@ redirects:
...
-4. Continue with the following procedure to embed the image in a doc.
+1. Place your image in the relevant directory. Give the image a descriptive file name; for example, `fso-ui-overview.png` is much better than `123-go-dawgs.png`.
+
+2. Continue with the following procedure to embed the image in a doc.
 
 ## Embed an image [#embed]
 
-Use markdown to embed an image in a doc. The basic structure:
+<CollapserGroup>
+  <Collapser title={<>Embed in <InlineCode>.mdx</InlineCode> file</>}>
```
Comment (josephgregoryii) [inline src/content/docs/style-guide/images/embed-images.mdx:59]: Not sure what is happening here, but in the readme preview on github its showing "Embed in .mdx file}>"

File: src/content/docs/style-guide/images/embed-images.mdx [hunk 3]
```diff
@@ -15,61 +15,114 @@ redirects:
...
+      src={syntheticsUiOverview}
+      title="Synthetics UI overview"
+      alt="An image showing an overview of the synthetic monitoring UI"
+    />
+
+    Rest of file
+    ```
+  </Collapser>
+
+  <Collapser title={<>Embed in <InlineCode>.md</InlineCode> file</>}>
```
Comment (josephgregoryii) [inline src/content/docs/style-guide/images/embed-images.mdx:79]: Same here

**Review by josephgregoryii — COMMENTED** [review]
Everything looks good from `eng` perspective. addressed some comments but feel free to dismiss them if they work on a build

File: src/content/docs/style-guide/images/embed-images.mdx (hunk 2 above)
Comment (roadlittledawn) [reply src/content/docs/style-guide/images/embed-images.mdx:59]: looks as expected when running locally

**Review by josephgregoryii — APPROVED** [review]
Thanks for confirming :shipit: 😃

File: src/content/docs/style-guide/images/embed-images.mdx [hunk 4]
```diff
@@ -15,61 +15,114 @@ redirects:
...
+  - title: Image codemod README
+    url: 'https://github.com/newrelic/docs-website/blob/b43d853f9ba4b17d4c090fb0a650b8f18cb0841d/codemods/convertMDXImages/README.md'
 ---
 
 A well-chosen screenshot or image can greatly improve the readability and clarity of a doc. Too many images or an image that's tough to parse can really slow things down.
 
-* If you're not part of the docs team and you want to add an image to the docs site, create a [GitHub issue](https://github.com/newrelic/docs-website/issues/new/choose).
-* If you're a New Relic employee, contact the `@hero` in the documentation Slack channel.
+- If you're not part of the docs team and you want to add an image to the docs site, create a [GitHub issue](https://github.com/newrelic/docs-website/issues/new/choose).
+- If you're a New Relic employee, contact the `@hero` in the documentation Slack channel.
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/images/embed-images.mdx:26]: We changed the name to help-documentation recently

File: src/content/docs/style-guide/images/embed-images.mdx [hunk 5]
```diff
@@ -15,61 +15,114 @@ redirects:
...
-* For screenshot captions, include the path in bold.
-* For video captions, include the approximate running time.
+- Make sure your image provides useful information at a glance.
+- Use alphanumeric characters in combo with `-` or `_` in the image file name. Avoid using any other special characters.
+- Include a caption with helpful context for the image.
+- For screenshot captions, include the path in bold.
+- For video captions, include the approximate running time.
 
-Our doc site images are stored in individual `images` directories at the root level of each taxonomy category. These `images` directories contain all the images used in the docs for that category. Do not link to an external file, all images used in docs must be stored in [our GitHub repo](https://github.com/newrelic/docs-website).
+Do not link to an external file, all images used in docs must be stored in [our GitHub repo](https://github.com/newrelic/docs-website).
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/images/embed-images.mdx:40]: I don't think a comma here is grammatically correct. Recommend change to a period "...file. All images..." or else a semi-colon.

File: src/content/docs/style-guide/images/embed-images.mdx [hunk 6]
```diff
@@ -15,61 +15,114 @@ redirects:
...
 We want to ensure that sensitive data does not appear in UI screenshots. That's why we recommend that you provide a permalink to a test account. Do not take a screenshot from your New Relic account.
 
 Here are some things to keep in mind when you're creating an image:
 
-* Make sure your image provides useful information at a glance.
-* Include a caption with helpful context for the image.
-* For screenshot captions, include the path in bold.
-* For video captions, include the approximate running time.
+- Make sure your image provides useful information at a glance.
+- Use alphanumeric characters in combo with `-` or `_` in the image file name. Avoid using any other special characters.
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/images/embed-images.mdx:35]: Should "combo" be "combination" or is casual "combo" ok here?

File: src/content/docs/style-guide/images/embed-images.mdx [hunk 7]
```diff
@@ -15,61 +15,114 @@ redirects:
...
+1. Place your image in the relevant directory. Give the image a descriptive file name; for example, `fso-ui-overview.png` is much better than `123-go-dawgs.png`.
+
+2. Continue with the following procedure to embed the image in a doc.
 
 ## Embed an image [#embed]
 
-Use markdown to embed an image in a doc. The basic structure:
+<CollapserGroup>
+  <Collapser title={<>Embed in <InlineCode>.mdx</InlineCode> file</>}>
+    Import your image in an `import` statement near the top of the MDX file. Ensure there are new lines above and below the import. The basic structure:
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/images/embed-images.mdx:60]: previously "MDX" was l/c ".mdx"

File: src/content/docs/style-guide/images/embed-images.mdx [hunk 8]
```diff
@@ -15,61 +15,114 @@ redirects:
...
 
-```
-![alt text](<var>PATH_TO_IMAGE</var> "Image title text")
-```
+    ```
+    ---
+    title: 'Example MDX file'
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/images/embed-images.mdx:64]: Previously in this doc MDX was l/c ".mdx"

File: src/content/docs/style-guide/images/embed-images.mdx [hunk 9]
```diff
@@ -78,50 +131,44 @@ Descriptive captions help the reader know why the image matters. If it's a scree
 
 For more help with captions and other supporting text around images, see [Guidelines for explaining images](/docs/style-guide/writing-guidelines/screenshots-images#identify-images).
 
-## Add an inline image [#inline-images]
+## Add an inline image (MDX) [#inline-images-mdx]
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/images/embed-images.mdx:134]: same comment re: MDX l/c

File: src/content/docs/style-guide/images/embed-images.mdx [hunk 10]
```diff
@@ -78,50 +131,44 @@ Descriptive captions help the reader know why the image matters. If it's a scree
...
 
-To use an inline image, add something like this:
+To use an inline image:
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/images/embed-images.mdx:136]: I think each bullets s/be initial cap and end with period.

File: src/content/docs/style-guide/images/embed-images.mdx [hunk 11]
```diff
@@ -78,50 +131,44 @@ Descriptive captions help the reader know why the image matters. If it's a scree
...
+* import the image at the top of the file
+* use the `img` tag in the file
+* add a class of `inline` to `img`
+* add any optional height/width in the `style` prop
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/images/embed-images.mdx:140]: Not sue what "prop" means. Property?

File: src/content/docs/style-guide/images/embed-images.mdx [hunk 12]
```diff
@@ -78,50 +131,44 @@ Descriptive captions help the reader know why the image matters. If it's a scree
...
 
-```
-<div style="width: 100%; max-width: Npx;">
-  <img alt="ALT TEXT" height="X" src="IMG_URL" title="FILENAME" width="N">
-</div>
-	
-<div class="dnd-legend-wrapper" style="width: 100%; max-width: Npx;">
-  <div class="meta"><p>CAPTION TEXT</p></div>
-</div>
+Select your app's settings <img src={exampleImage} class="inline" style={{height: "30px", width: "30px"}} alt="Multiple app names indicator in New Relic One" title="Multiple apps name indicator in New Relic One" /> icon.
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/images/embed-images.mdx:157]: This is SO helpful!!

File: src/content/docs/style-guide/images/embed-images.mdx [hunk 13]
```diff
@@ -78,50 +131,44 @@ Descriptive captions help the reader know why the image matters. If it's a scree
...
 
 ## Icons [#icons]
 
-You can choose from a variety of icons to include in your docs: 
+You can choose from a variety of icons to include in your docs:
 
-* Feather icons (prefixed with `'fe-`), which replace our previous Font Awesome icons
-* New Relic icons (prefixed with `nr-`)
-* Logos for third-party products (prefixed with `logo-`)
+- Feather icons (prefixed with `'fe-`), which replace our previous Font Awesome icons
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/images/embed-images.mdx:164]: Do we need to refer to Font Awesome anymore?

**Review by barbnewrelic — CHANGES_REQUESTED** [review]
This is really helpful info, thanks so much! I am wondering if @rhetoric101 might also want to review since he has worked a lot on the image docs in the past?

File: src/content/docs/style-guide/images/embed-images.mdx (hunk 13 above)
Comment (roadlittledawn) [reply src/content/docs/style-guide/images/embed-images.mdx:164]: 🤔  probably not?

File: src/content/docs/style-guide/images/embed-images.mdx (hunk 11 above)
Comment (roadlittledawn) [reply src/content/docs/style-guide/images/embed-images.mdx:140]: props (in JSX) or attributes (in HTML) would things like `class`, `style`, `alt`, `title` in the tag like so:

```jsx
<img alt="text" title="text" style={{height: '29px'}} class="inline" />
```

**Pull request merged**